html-forms/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form action="index.html" method="post">
+      <div><label for="email">Email</label></div>
+      <input id="email" type="text" name="email">
+      </form>
+
+    <form action="index.html" method="post">
+      <div>Why is coding cool?</div>
+      <select name="reason">
+        <option id="select" value="default">Select an option</option>
+        <label>Select an option</label>
+        <option id="interesting" value="interesting">Because it's interesting</option>
+        <label>Because it's interesting</label>
+        <option id="fun" value="fun">Because it's fun</option>
+        <label>Because it's fun</label>
+        <option id="powerful" value="powerful">Because it's powerful</option>
+        <label>Because it's powerful</label>
+      </select>
+    </form>
+
+    <form action="index.html" method="post">
+      <div>Favorite Language (choose one)</div>
+        <input id="html" type="radio" name="language" value="HTML">
+        <label for="html">HTML</label>
+        <input id="css" type="radio" name="language" value="CSS">
+        <label for="css">CSS</label>
+        <input id="javascript" type="radio" name="language" value="JavaScript">
+        <label for="javascript">JavaScript</label>
+    </form>
+
+    <form action="index.html" method="post">
+      <div>Would you like to sign up for more information?</div>
+        <input id="signup" type="checkbox" name="service" value="yes">
+        <label for="signup">Sign me up!</label>
+    </form>
+
+    <form action="index.html" method="post">
+      <div>Comments</div>
```

Comment:
This should be the `label` element, not a `div`. This is the text your user would click to try and put focus on the input it pairs with.